spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-event-tracker.spec.js

```diff
@@ -462,4 +462,49 @@ describe('Google Analytics event tracking', function () {
       expect(window.dataLayer[0]).toEqual(expected)
     })
   })
+
+  describe('doing tracking on the super navigation header', function () {
```

Comment:
Following on my comment about the `target.closest` line, one benefit would be that we could make this section of testing more generic, and not have to add specific tests for each kind of new thing.

Reply:
Thanks @andysellick - I renamed the test to make it more generic, I wasn't sure how to change the test itself to make it more generic but let me know if you have an idea.

Reply:
I guess we can do that when we remove the explicit accordion class, so fine for now.